.babelrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
if we're gonna use babelrc we should move all babel config out of webpack.config.js